### DISCUSSION TITLE
shell unusable buggy

### DISCUSSION THREAD

**Discussion by akomm** [oven-sh/bun#13121] [General]
Should be documented properly as alpha stage. I've hit uncountable amount of bugs with arg processing, be it running scripts in package.json, or simply passing empty args like `-X ""` the empty args being ignored and many more. Some of the bugs are plain dangerous.

My plan on using it on a new cli app are dropped for the mid-long term future :(.

**Comment by staabm** [comment]
You should provide reproducible snippets of what does not work and what you expect instead. 

The discussion you just opened does not provide value for anyone.

Even people willing to help investigate your problem don't have enough information to fix the cause of your issues

**Reply by SWATOPLUS** [reply]
I think issues already reported here https://github.com/oven-sh/bun/issues/13316

**Reply by akomm** [reply]
Yes if it was not reported already, I would have created an issue. I merely note it here as a discussion, that it should be marked as unstable at its current state. And maybe overthink the way how the team determines APIs stable or not. It does unnecessary create bad impression, when you hit this sort of frequent issues, except it was clearly communicated as unstable, then its own risk.
